starboard/linux/shared/cobalt/configuration.py

```diff
@@ -61,6 +63,11 @@
     ],
 }
 
+if os.getenv('MODULAR_BUILD', '0') == '1':
```

Comment:
where is this value set in buildbot ?

I dont see it being set in the modular build runner: https://github.com/youtube/cobalt/actions/runs/5722805184/job/15506464207?pr=618

Reply:
https://github.com/youtube/cobalt/blob/main/.github/workflows/main.yaml#L281